include/wpe/version.h.cmake

```diff
@@ -31,6 +31,16 @@
 #ifndef wpe_version_h
 #define wpe_version_h
 
+/**
+ * SECTION:version
+ * @short_description: Library Version
+ * @title: Version
+ * @symbols:
+ *   - wpe_backend_get_major_version
```

Comment:
These symbols *need* to be listed, because I want *all* the versioning-related symbols (even these deprecated ones!) to be in the same page of the documentation. Not listing them here causes a new “version deprecated” page to be created from the generated `wpe/version-deprecated.h` header.

Reply:
Indeed, I misread, nvm :-)